e2e-tests/cypress/e2e/journeys/insurance/account/create/confirm-email/confirm-email.spec.js

```diff
@@ -0,0 +1,131 @@
+  });
+
+  it('passes the audits', () => {
+    cy.lighthouse({
+      accessibility: 100,
+      performance: 75,
+      'best-practices': 100,
+      seo: 70,
+    });
+  });
+
+  it('renders a back link with correct url', () => {
+    partials.backLink().should('exist');
+    cy.checkText(partials.backLink(), LINKS.BACK);
+
+    partials.backLink().click();
+
+    const expectedUrl = `${Cypress.config('baseUrl')}${YOUR_DETAILS}`;
+
+    cy.url().should('eq', expectedUrl);
+
+    // go back to page
+    cy.navigateToUrl(CONFIRM_EMAIL);
+  });
+
+  it('renders an analytics cookies consent banner that can be accepted', () => {
+    cy.checkAnalyticsCookiesConsentAndAccept();
+  });
+
+  it('renders an analytics cookies consent banner that can be rejected', () => {
+    cy.rejectAnalyticsCookies();
+  });
+
+  it('should render a header with href to insurance start', () => {
+    partials.header.serviceName().should('have.attr', 'href', START);
+  });
+
+  it('renders a phase banner', () => {
+    cy.checkPhaseBanner();
+  });
+
+  it('renders a page title and heading', () => {
+    const expectedPageTitle = `${CONTENT_STRINGS.PAGE_TITLE} - ${ORGANISATION}`;
+    cy.title().should('eq', expectedPageTitle);
+
+    cy.checkText(heading(), CONTENT_STRINGS.PAGE_TITLE);
+  });
+
+  it('renders `sent a link to`', () => {
+    const expected = `${CONTENT_STRINGS.WE_SENT_LINK_TO} TODO`;
+
+    cy.checkText(confirmEmailPage.weSentLinkTo(), expected);
+  });
+
+  it('renders `check your email`', () => {
+    const expected = CONTENT_STRINGS.CHECK_YOUR_EMAIL;
+
+    cy.checkText(confirmEmailPage.checkYourEmail(), expected);
+  });
+
+  describe('having problems section', () => {
+    const STRINGS = CONTENT_STRINGS.HAVING_PROBLEMS;
+    const { wrongEmail } = confirmEmailPage.havingProblems;
+
+    it('renders a heading', () => {
+      const expected = STRINGS.HEADING;
+
+      cy.checkText(confirmEmailPage.havingProblems.heading(), expected);
+    });
+
+    it('renders `can request a new link`', () => {
+      const { REQUEST_NEW } = STRINGS;
+      const { YOU_CAN, LINK, IF_NOT_RECEIVED } = REQUEST_NEW;
+
+      cy.checkText(confirmEmailPage.havingProblems.requestNew.youCan(), YOU_CAN);
+
+      cy.checkText(confirmEmailPage.havingProblems.requestNew.link(), LINK.TEXT);
+      confirmEmailPage.havingProblems.requestNew.link().should('have.attr', 'href', LINK.HREF);
+
+      cy.checkText(confirmEmailPage.havingProblems.requestNew.ifNotReceived(), IF_NOT_RECEIVED);
+    });
+
+    it('renders `if entered wrong email`', () => {
+      const { WRONG_EMAIL } = STRINGS;
+      const { ENTERED_INCORRECTLY, CREATE_ACCOUNT_AGAIN } = WRONG_EMAIL;
+
+      cy.checkText(confirmEmailPage.havingProblems.wrongEmail.enteredIncorrectly(), ENTERED_INCORRECTLY);
+
+      cy.checkText(wrongEmail.createAccountAgainLink(), CREATE_ACCOUNT_AGAIN.TEXT);
+      wrongEmail.createAccountAgainLink().should('have.attr', 'href', ROUTES.ACCOUNT.CREATE.YOUR_DETAILS);
```

Comment:
can we make this a cy command? tech debt for later

Reply:
I thought about this as well - added to the list of tickets to create, am thinking we can have `cy.checkLink` which tests the text and the href